client/my-sites/email/email-providers-comparison/stacked/index.tsx

```diff
@@ -269,6 +325,35 @@ const EmailProvidersStackedComparison = ( {
 			{ ! isDomainInCart && domain && <EmailExistingPaidServiceNotice domain={ domain } /> }
 
 			<>
+				{ ! currentUserCanAddEmail && (
+					<PromoCard className="email-providers-stacked-comparison__owner-notice">
```

Comment:
Wondering if it makes more sense for us to rename the class to `email-providers-stacked-comparison__nonowner-notice`, since this component is shown to non-owners.